meta/main.yml

```diff
@@ -8,3 +8,4 @@ galaxy_info:
     - name: Debian
       versions:
         - stretch
+        - buster
```

Comment:
In order to ensure that is working under stretch and buster, you should create a job matrix in Travis config file setting MOLECULE_DISTRO to stretch or buster in each execution:

https://github.com/idealista/airflow-role/blob/feature/add-python-3.8-and-venv/.travis.yml